lua/lspconfig/server_configurations/hhvm.lua

```diff
@@ -0,0 +1,27 @@
+local util = require 'lspconfig.util'
+
+local bin_name = 'hh_client'
```

Comment:
There's no reason for these not to be defined in the table, plus the way I did autodocgen it will insert `cmd` automatically if it is inline (using the lua debug module)

Reply:
Thanks, inlined them in the following commit